doc-surrealdb_versioned_docs/version-1.x/security/authentication.mdx

```diff
@@ -240,4 +240,18 @@ curl -X POST \
 	-H "Accept: application/json" \
 	-d '{"NS":"test", "DB":"test", "SC":"user", "email":"john@doe.org", "password":"VerySecurePassword!"}' \
 	http://localhost:8000/signin
-```
\ No newline at end of file
+```
+
+## Sessions
+
+Whenever authentication is performed with any kind of user against SurrealDB, a session is established between the client and the SurrealDB server with which the connection was established. These sessions persist on the server for the duration of the connection, whether it is a single request through the [HTTP REST API](/docs/surrealdb/integration/http) or through multiple requests in the same connection using the [WebSocket API](/docs/surrealdb/integration/rpc) and any of the [SDK](/docs/surrealdb/integration/sdks/) that leverage it.
+
+### Expiration <l className='purple'>Since 1.3.0</l>
+
+Authenticated sessions established using a token (i.e. JWT) will remain authenticated for as long as that token is valid according to its `exp` claim. Whenever a token that was used to establish a session expires, so will the session. This is specially relevant in the case of [WebSocket](/docs/surrealdb/integration/rpc) connections such as those used by the [SurrealDB SDK](/docs/surrealdb/integration/sdks/). Expired sessions can no longer be used to call authenticated methods and doing so will result in a specific error indicating that the session has expired.
```

Comment:
1) Recommend changing specially to especially
2) Probably change i.e. to e.g. (assuming that JWT is an example of a token, not that tokens *are* JWT)
3) I'm curious, what exactly does it mean to call authenticated methods? This sort of looks to me as that some actions can still be done when a session is expired but I'm not sure if I'm reading it right. If a session is expired can a client do anything else besides using a new token as mentioned in the next paragraph?

Reply:
1. Nice catch! I will change it.
2. Tokens are always JWT, hence the use of "i.e.". I wanted to avoid saying "JWT token" as that is what the "T" stands for.
3. Yes, some actions can be done with an anonymous session. These actions include `ping`, `use`, `invalidate`, `version`, `authenticate`, `signin` and `signup`. They are detailed in the tests but I did not think this was the place to describe such a technicality that is subject to change in the future.

Reply:
Oh yeah, that could be a bit too much information. Agreed!